test/test_fx_experimental.py

```diff
@@ -1482,12 +1482,9 @@ def test_normalize_operator_exhaustive(self, device, dtype, op):
             "igammac",
             "index_put",
             "nn.functional.conv2d",
-            "nn.functional.dropout",
-            "nn.functional.dropout2d",
             "nn.functional.embedding",  # Implemented with a lambda
             "nn.functional.embedding_bag",  # Implemented with a lambda
             "nn.functional.rrelu",  # Implemented with a lambda
-            "nn.functional.feature_alpha_dropout",  # Implemented with a lambda
```

Comment:
The dropout variants have been shifted to having an expected failure in their respective OpInfos.

Reply:
Nice!